media_driver/linux/common/os/i915/mos_bufmgr.c

```diff
@@ -977,7 +977,7 @@ mos_gem_bo_alloc_internal(struct mos_bufmgr *bufmgr,
     pthread_mutex_unlock(&bufmgr_gem->lock);
 
     if (!alloc_from_cache) {
-        struct drm_i915_gem_create create;
+        struct drm_i915_gem_create_android create;
```

Comment:
Here is a grep on the code:
```
[dvrogozh@dvrscl media-driver]$ fgrep -rsn drm_i915_gem_create_android
[dvrogozh@dvrscl media-driver]$
```
As you see, there is no drm_i915_gem_create_android, thus this code simply will not compile.